packages/core/addon/models/fragments/scheduling-rules.ts

```diff
@@ -8,6 +8,9 @@ import Fragment from 'ember-data-model-fragments/fragment';
 export default class SchedulingRuleFragment extends Fragment {
   @attr('boolean')
   mustHaveData!: boolean;
+
+  @attr('boolean')
+  overwriteFile!: boolean;
```

Comment:
IMHO, I think this could be better placed as an option of the format, as not all formats will support it (at least initially)
```ts
{
  "format": {
    "type": "gsheet",
    "options": {
      "overwriteFile": true
    }
  }
}
```

This could trail blaze the ability to add more options for particulars format in the future.

Another thought that comes to mind is that the original idea behind `scheduling-rules` is that it contains the rules for when to kick off a delivery. Not sure `overwriteFile` fits that idea.